src/app/core/utils/math/BezierCurve.ts

```diff
@@ -3,16 +3,59 @@ import {Transform} from "./Transform";
 import {Clamp} from "./MathUtils";
 import {serializable} from "serialeazy";
 
-@serializable("BezierCurve")
+/**
```

Comment:
nit: should have two empty lines between last import and the start of the comment